docs/swagger/spec.yml

```diff
@@ -466,32 +472,70 @@ paths:
         - $ref: '#/parameters/minSunAzimuth'
         - $ref: '#/parameters/maxSunElevation'
         - $ref: '#/parameters/minSunElevation'
-        - $ref: '#/parameters/tags'          
+        - $ref: '#/parameters/tags'
       responses:
         200:
           description: Paginated list of scenes associated with this bucket
           schema:
             $ref: '#/definitions/ScenePaginated'
     post:
-      summary: Create a new association between a scene and this bucket
+      summary: Create a new association between a set of scenes and this bucket
       tags:
         - Imagery
       parameters:
         - $ref: '#/parameters/uuid'
-        - name: image
+        - name: scenes
           in: body
           required: true
+          description: UUIDs of scenes to replace in bucket
           schema:
-            $ref: '#/definitions/Scene'
+            type: array
+            items:
+              type: string
+              format: uuid
       responses:
-        202:
-          description: Bucket image details; thumbnails and boundary may be in-progress
+        201:
+          description: List of scenes added to bucket
           schema:
-            $ref: '#/definitions/Image'
-        200:
-          description: Bucket image details; the image already existed
+            type: array
+            items:
+              $ref: '#/definitions/Scene'
+    put:
+      summary: Replace the set of scenes in a bucket
+      tags:
+        - Imagery
+      parameters:
+        - $ref: '#/parameters/uuid'
+        - name: scenes
+          in: body
+          required: true
+          description: UUIDs of scenes to delete in bucket
           schema:
-            $ref: '#/definitions/Image'
+            type: array
+            items:
+              type: string
+              format: uuid
+      responses:
+        204:
+          description: No content, update successful
+    delete:
+      summary: Delete a set of scenes from a bucket
+      tags:
+        - Imagery
+      parameters:
+        - $ref: '#/parameters/uuid'
+        - name: scenes
+          in: body
+          required: true
+          description: UUIDs of scenes to delete in bucket
+          schema:
+            type: array
+            items:
+              type: string
+              format: uuid
+      responses:
+        204:
+          description: No content, delete successful
   /buckets/{uuid}/scenes/{sceneUUID}:
```

Comment:
The individual delete endpoint is being removed (to delete 1 scene, just pass /buckets/{uuid}/scenes a list with a single scene id